test/hotspot/jtreg/vmTestbase/nsk/share/jpda/DebugeeProcess.java

```diff
@@ -326,12 +327,19 @@ public void redirectStdout(OutputStream out) {
      * was already redirected, the TestBug exception is thrown.
      */
     public void redirectStdout(Log log, String prefix) {
+        redirectStdout(log, prefix, null);
```

Comment:
Is there a reason to still support passing in the prefix even though it is ignored. Seems that could be confusing to someone looking at the caller code.

Reply:
I dont' quite follow.
The prefix is not ignored, it's passed to IORedirector and is used for logging

Reply:
Sorry, I misread it. I thought the 3rd argument passed was the prefix.